kube-runtime/src/controller/mod.rs

```diff
@@ -593,6 +593,55 @@ where
     /// The [`ListParams`] refer to the possible subset of `Watched` objects that you want the [`Api`]
     /// to watch - in the Api's configured scope - and run through the custom mapper.
     /// To watch the full set of `Watched` objects in given the `Api` scope, you can use [`ListParams::default`].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use kube::runtime::{Controller, controller::Action, reflector::ObjectRef};
+    /// # use kube::api::{Api, ListParams};
+    /// # use kube::ResourceExt;
+    /// # use k8s_openapi::api::core::v1::{ConfigMap, Namespace};
+    /// # use futures::StreamExt;
+    /// # use std::sync::Arc;
+    /// # type ApiResource = Namespace;
```

Comment:
This hidden `type = ` is confusing here because it is shadowing an existing type. Let's kill this line and use `Api::<Namespace>` below 
```suggestion
```

Reply:
I also commented below, I created the alias so the generic name `ApiResource` can be used in the visible documentation. It would not make much sense in the example to have a controller for config maps which watches namespaces imo.

Reply:
That makes sense. I am more worried about people confusing it with [`ApiResource`](https://docs.rs/kube/latest/kube/api/struct.ApiResource.html) which has slightly more subtle dynamic properties, whereas it can really mean "any Resource" here.

Maybe we can name it something more obvious in that case? E.g. `SomeResource` / `WatchedResource` / `WatchedK`?

Reply:
I updated it to `WatchedResource`, I feel like that works best with the references to `Watched` in the docs above